indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -118,6 +129,76 @@ class BookingDetails extends React.Component {
     );
   };
 
+  renderPreBookingWarning = () => {
+    const {bookingStateChangeInProgress} = this.props;
+    const {
+      acceptanceData,
+      actionInProgress,
+      preBookingWarningVisible,
+      concurrentBookings,
+    } = this.state;
+    const items = concurrentBookings.map(booking => {
+      const reservation = booking.reservation;
+      const startDate = serializeDate(toMoment(booking.start_dt), 'L LT');
+      const endDate = serializeDate(toMoment(booking.end_dt), 'LT');
+      return (
+        <List.Item key={reservation.id}>
+          <a
+            href={bookingLinkURL({booking_id: reservation.id})}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {startDate} - {endDate}
+          </a>
+          <Translate>
+            {' '}
+            by <Param name="user" value={reservation.booked_for_name} /> {'('}
+            <Param name="info" value={reservation.booking_reason} />
+            {')'}
+          </Translate>
```

Comment:
Don't put leading/trailing whitespace inside Translate tags - it's not pleasant when translating and I think we even strip it somewhere.

```suggestion
          </a>{' '}
          <Translate>
            by <Param name="user" value={reservation.booked_for_name} /> (
            <Param name="info" value={reservation.booking_reason} />)
          </Translate>
```

In principle you could even move the ` (reason)` part outside the translate block, since i don't think any language needs to change the order of name and reason:

```jsx
          </a>{' '}
          <Translate>
            by <Param name="user" value={reservation.booked_for_name} />
          </Translate>{' '}
          ({reservation.booking_reason})
```